src/opentimelineio/CMakeLists.txt

```diff
@@ -78,14 +78,16 @@ add_library(opentimelineio ${OTIO_SHARED_OR_STATIC_LIB}
 add_library(OTIO::opentimelineio ALIAS opentimelineio)
 
 target_include_directories(opentimelineio
-    PRIVATE       "${IMATH_INCLUDES}"
-                  "${PROJECT_SOURCE_DIR}/src"
+    PRIVATE       "${PROJECT_SOURCE_DIR}/src"
                   "${PROJECT_SOURCE_DIR}/src/deps"
-                  "${PROJECT_SOURCE_DIR}/src/deps/rapidjson/include"
-                  "${IMATH_INCLUDES}")
+                  "${PROJECT_SOURCE_DIR}/src/deps/rapidjson/include")
+
+if(USE_DEPS_IMATH)
+    target_include_directories(opentimelineio PRIVATE "${PROJECT_SOURCE_DIR}/src/deps/Imath/src")
```

Comment:
I know `PRIVATE` was from the original code, but should it be removed since Imath is part of the public API? Would that allow other targets using the `opentimelineio` target to pick up the Imath dependency? Like the Python bindings, tests, or if OTIO is being used as a Git sub-module.

Reply:
you are correct, it needs to be public since the imath types appear in interfaces

Reply:
I am happy with whatever you prefer

Reply:
@darbyjohnston you are saying this pattern avoids the prefixing problem? (I think that it would) `include_directories("${PROJECT_SOURCE_DIR}/src/deps/Imath/src")`

Reply:
I think so, it would also simplify the changes a bit by removing the various calls to "target_include_directories".

I'm not sure why the "Imath::Imath" "INTERFACE_INCLUDE_DIRECTORIES" don't handle the prefix correctly, is that something that needs to be fixed on the Imath side? Or something about how OTIO is using Imath as a sub-module?

@jlskuz Do you mind trying this change at the top-level CMakeLists.txt, and removing the "target_include_directories" calls? Otherwise the changes look good, thanks for the help updating this.
```
if(USE_DEPS_IMATH)
    include_directories("${PROJECT_SOURCE_DIR}/src/deps/Imath/src")
endif()
```

Reply:
Done

Reply:
Thanks for the changes!

@meshula Just confirming that we are OK with dropping support for Imath2? Or does that need to be a larger conversation?